dashboard/config/levels/custom/spritelab/csc_landmarks_behavior_2023.level

```diff
@@ -50,60 +54,61 @@
   "level_concept_difficulty": {
   }
 }]]></config>
   <blocks>
     <start_blocks>
-      <xml>
-        <block type="gamelab_behavior_get" uservisible="false">
-          <mutation/>
-          <field name="VAR" id="moving NESW with arrow keys">moving NESW with arrow keys</field>
-        </block>
-        <block type="when_run" deletable="false" movable="false">
+      <xml xmlns="https://developers.google.com/blockly/xml">
```

Comment:
This shows that the start blocks were saved with Google Blockly, fyi @fisher-alice